### ISSUE TITLE
Pour mardi 7 fevrier 2017.

### ISSUE THREAD

**Issue by ThomasGire** [EdisonLorgues1SD1617/Devoirs#40]
1. Exercices [20 à 22 page 304](https://raw.githubusercontent.com/EdisonLorgues1SD1617/Math1SD1617/master/Donn%C3%A9es/Chapitres/6.%20Probabilit%C3%A9s/Images/20-26p304.png): Calculer les paramètres d'une loi de probabilité.

@EdisonLorgues1SD1617/classe

- [ ] Alexandre Bouille
- [ ] Allan Hourdin
- [ ] Antoni Gagno
- [ ] Aurore Charles
- [ ] Bassir El Ouaadi
- [ ] Camille Lecour
- [ ] Cédric Prast
- [ ] Eliot Gonnet
- [ ] Fabien Viazzi
- [ ] Grégory Faitot
- [ ] Guillaume Delattre
- [ ] Guillaume Raynaud
- [ ] Gwendal Jean
- [ ] Henry Jugy
- [ ] Kellian Costa
- [ ] Louis Hassler
- [ ] Louis Vilain
- [ ] Ludovic Nunes Da Silva
- [ ] Marion Deisard-Decot
- [ ] Mary Michel
- [ ] Mathilde Lacroix
- [ ] Maximilian Riepl
- [ ] Mohamed Ayadi
- [ ] Nicolas Doussin
- [ ] Noe Conti
- [ ] Olivier Piollet
- [ ] Thomas Slack
- [ ] Virgil Couturier
- [ ] William Fernandes
- [ ] Yanis Galland

**Comment by MaryMichel** [comment]
https://github.com/MaryMichel/20-22p304

**Comment by MaximilianRiepl** [comment]
https://github.com/MaximilianRiepl/Ressources/blob/master/img002.jpg
https://github.com/MaximilianRiepl/Ressources/blob/master/img003.jpg

**Comment by GwendalJean** [comment]
Mary je crois que tu as faux pour la dernière question de l'ex 21, le jeu est équitable.

Voila mes exercices
https://docs.google.com/document/d/1jeIWblPReCcBmEDppbRtyL-jYxUWzD8HsYpAfTcq1ZI/edit

**Comment by AllanHourdin** [comment]
Je suis d'accord avec Gwendal , le jeu est équitable ( je partagerai demain ) 

Gwendal , j'arrive pas a voir tes exos ( tu es sur google drive ???)

**Comment by MarionDeissard-Decot** [comment]
![20170206_174924](https://cloud.githubusercontent.com/assets/21561038/22657230/0bf33006-ec96-11e6-8d5b-c42b807fac37.jpg)

**Comment by AllanHourdin** [comment]
![img_20170206_180215](https://cloud.githubusercontent.com/assets/21603124/22657575/46ca9ac4-ec97-11e6-9c17-3e97fc862e94.jpg)

**Comment by MohamedAyadi1** [comment]
Monsieur voilà le Dm: https://github.com/MohamedAyadi1/TravailPersonnel/blob/master/DSC_0006.JPG je fais les exercices ensuite

**Comment by AllanHourdin** [comment]
Mohamed : Ton dm à l'air juste

2017-02-06 18:40 GMT+01:00 MohamedAyadi1 <notifications@github.com>:

**Comment by BassirElOuaadi** [comment]
![wp_20170206_001](https://cloud.githubusercontent.com/assets/21574521/22659935/e2ebf080-ec9f-11e6-837c-15e0ada7ed7e.jpg)
![wp_20170206_003](https://cloud.githubusercontent.com/assets/21574521/22659937/e2ed4598-ec9f-11e6-90bc-ff55683ed999.jpg)
![wp_20170206_005](https://cloud.githubusercontent.com/assets/21574521/22659936/e2ec5a8e-ec9f-11e6-86ff-523404528ee3.jpg)

**Comment by MathildeLacroix** [comment]
https://github.com/MathildeLacroix/TravailPersonnel/blob/master/Exo20%C3%A022p304.jpg

**Comment by LouisVilain** [comment]
https://fr.sharelatex.com/project/5898bf02a543fec64a5368ca
voila mon travail (realisé sur ShareLatex)

**Comment by HenryJugy** [comment]
![image](https://cloud.githubusercontent.com/assets/21668386/22668493/30c36efc-ecc1-11e6-9570-ceaedb0cb293.jpg)

**Comment by WilliamFernandes1** [comment]
https://github.com/WilliamFernandes1/exercices/blob/master/IMG_1816.JPG
